### PR TITLE
Delete pr-validation-temp.yml

### PR DESCRIPTION
**Pull request by wtgodbe**
This is no longer needed now that 2.1 builds in AzDO

### CODE REVIEW

**Comment by wtgodbe** [comment]
I also changed the required check from pr-validation-temp to aspnetcore-ci

**Comment by dougbu** [comment]
> I also changed the required check from pr-validation-temp to aspnetcore-ci

Think the release/2.1 protection rule can just be deleted. Looks like it matches the `release/*` rule. Right❔

**Comment by wtgodbe** [comment]
> Think the release/2.1 protection rule can just be deleted. Looks like it matches the release/* rule. Right❔

Agreed, looks the same to me. I just deleted it.

File: .azure/pipelines/pr-validation-temp.yml [hunk 1]
```diff
@@ -1,38 +0,0 @@
...
-    include:
-    - '*'
-
-jobs:
-- template: jobs/default-build.yml
-  parameters:
-    jobName: Windows_Build
-    jobDisplayName: "Build only : Windows"
-    agentOs: Windows
-    buildArgs: '/p:SkipTests=true'
```
Comment (dougbu) [inline .azure/pipelines/pr-validation-temp.yml:18]: Note this was ignored anyhow. See https://dev.azure.com/dnceng/public/_build/results?buildId=969282 and the warnings about `$(BuildScriptArgs)`. Might have a bug in this branch's version of default-build.yml

**Review by dougbu — APPROVED** [review]

**Comment by dougbu** [comment]
One other point: Once #29037 and anything else in-progress drains from aspnetcore-pr-validation-temp, please delete the pipeline too

File: .azure/pipelines/pr-validation-temp.yml (hunk 1 above)
Comment (dougbu) [reply .azure/pipelines/pr-validation-temp.yml:18]: Found it. I'll put up a quick PR later today to fix it.

Comment (dougbu) [reply .azure/pipelines/pr-validation-temp.yml:18]: #29737

**Pull request merged**